### ISSUE TITLE
Unexpected behavior when combining `initial` and `range` as iterator.

### ISSUE THREAD

**Issue by twall** [tqdm/tqdm#1561]
- [x] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [x] visual output bug
- [x] I have visited the [source website], and in particular
  read the [known issues]
- [x] I have searched through the [issue tracker] for duplicates
- [x] I have mentioned version numbers, operating system and
  environment, where applicable:
  ```python
  import tqdm, sys
  print(tqdm.__version__, sys.version, sys.platform)
  ```
4.66.2 3.10.9 (main, Jan 11 2023, 09:18:20) [Clang 14.0.6 ] darwin

[source website]: https://github.com/tqdm/tqdm/
[known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
[issue tracker]: https://github.com/tqdm/tqdm/issues?q=

```
from tqdm import tqdm 

for level in tqdm(list(range(1, 16)), initial=10):
  print(f"Level={level}")
```

Output:
```
 67%|████████████████████████           | 10/15 [00:00<?, ?it/s]
Level 1
Level 2
Level 3
Level 4
Level 5
Level 6
Level 7
Level 8
Level 9
Level 10
Level 11
Level 12
Level 13
Level 14
Level 15
```

In this case, the `initial` argument is not completely ignored, since the status seems to stop at `10`.  I would expect either an exception or warning about the argument being ignored, or in the case of an indexable iterable actually use it to start at the correct index.

**Comment by paoyw** [comment]
Hi @twall,

I have try to reproduce the issue here. The output is following
```
 67%|████████████████████████████▋              | 10/15 [00:00<?, ?it/s]Level=1
Level=2
Level=3
Level=4
Level=5
Level=6
Level=7
Level=8
Level=9
Level=10
Level=11
Level=12
Level=13
Level=14
Level=15
25it [00:00, 205603.14it/s]
```
I am not sure if the last line is missed in your post.
Based on [document](https://tqdm.github.io/docs/tqdm/) and the response(https://github.com/tqdm/tqdm/issues/215#issuecomment-235502551) to a similar issue, the `initial` only sets the initial offset for iteration display.
And I think the following code is what you are looking for.
```python
from tqdm import tqdm 

for level in tqdm(list(range(1, 16)), initial=10, total=10+15):
  print(f"Level={level}")
```

**Comment by twall** [comment]
I am expecting the iterations to _start_ at 10 and _finish_ at 15, for a total of 5 iterations.

**Comment by twall** [comment]
```
tqdm(list(range(10, 16)), initial=10, total=16)
```
_almost_ works, except the final status is 16/16 instead of 15/15.  Using `total=15` results in an static progress bar stuck at 10/15.

**Comment by paoyw** [comment]
Change the to `tqdm(list(10, 16), initial=9, total=15)` will meet your needs.
Based on my understanding, you want to iterate through `[1..15]`, which leaves `total=15` originally and want to keep on iterate though `[10..15]`.
It means that you have iterated through `[1..9]`, so the initial counter should be 9, or `initial=9` in the argument.

**Comment by twall** [comment]
Yes, that works.  I was mistakenly interpreting `initial` to mean the first iterable _value_, rather than the number of iterations _completed_.  `initial` really has nothing at all to do with the iterable itself.  It is a little odd having the status bar start at `9/15`, but that's consistent with `0/N` being the initial status of any other N count.
